docs/workshop/index.md

```diff
@@ -0,0 +1,410 @@
+---
+title: GeoMoose 3.0 Workshop
+author: Dan "Ducky" Little / @theduckylittle
+date: 15 August 2017
+version: 3.0.0
+output:
+    beamer_presentation:
+        slide_level: 2
+        fig_width: 5
+        fig_height: 3.5
+header-includes:
+  - \hypersetup{colorlinks=false,
+       allbordercolors={0 0 0},
+       pdfborderstyle={/S/U/W 1}}
+fontsize: 8pt
+---
+
+## About this workshop
+
+* Learn how to install GeoMoose, add data to the application, and customize the look and feel.
+* This workshop assumes the use of MS4W as the installation platform. GeoMoose is Linux/UNIX/OS-X
+  friendly and is, in fact, developed on Linux but the install on Linux is out of the scope
+  of this workshop.
+
+## Who is this workshop for?
+
+* Anyone wanting to have a WebGIS application for their users.
+* "GIS Admins" - People who have data and want to get it out.
+* Developers who don't want to repeat themselves.
+
+# Installing MS4W
+
+## Getting started with MS4W
+
+MapServer for Windows (MS4W) will install the base files required by GeoMoose along with an Apache service to actually serve GeoMoose. Once installed, navigate to localhost for more detail  Also, full description of optional packages in the installer are found on ms4w.com.
+
+Requirements:
+
+ * A computer running Windows 7 or newer.
+ * A copy of MS4W.  [Download Here.](http://ms4w.com)
+ * A copy of the GeoMoose Demo Data. [Download Here.](https://www.geomoose.org/downloads/gm3-demo-data-3.0.0-ms4w.zip)
+ * A copy of the GeoMoose MS4W Package [Download Here.](https://www.geomoose.org/downloads/gm3-examples-3.0.0-ms4w.zip)
+
+
+## Install MS4W
+
+**Double click the ms4w-3.2.2-setup.exe icon.**
+
+
+MapServer for Windows does come with an older version of GeoMoose, but it is recommended to skip that and install the latest and greatest version for the full Moose experience.
+
+![ms4w setup screen 1](../ms4w-quickstart/ms4w-setup-1.png)
+
+## Select "C:\\"
```

Comment:
Why "C:\\"  (The '\\' shows up in the PDF so I don't think it is treating '\' as an escape).